source/docs/software/driverstation/driver-station-labview-dashboard.rst

```diff
@@ -0,0 +1,127 @@
+- Commands tab for use with the new LabVIEW C&C Framework
+- Checklist tab that can be used to create task lists to complete before and/or between matches
+- Variables tab that displays the raw Network Tables variables in a tree view format
+
+The LabVIEW Dashboard also includes Record/Playback functionality, located in the bottom right. More detail about this feature is included below under `Record/Playback`_.
+
+Camera Image and Controls
+-------------------------
+
+.. image:: images/driver-station-labview-dashboard/camera-image-and-controls.png
+
+The left pane is used to display a video feed from an Axis camera or USB camera located on the robot. For instructions on setting up the Axis Camera to work with this display :ref:`see here <docs/software/vision-processing/introduction/configuring-an-axis-camera:Configuring an Axis Camera>`. There are also some controls and indicators related to the camera in the right pane on the Drive tab and below the tab area:
+
+#. Camera Image Display
+#. Mode Selector - This drop-down allows you to select the type of camera display to use. The choices are Camera Off, USB Camera SW (software compression), USB Camera HW (hardware compression) and IP Camera (Axis camera). Note that the IP Camera setting will not work when your PC is connected to the roboRIO over USB.
+#. Camera Settings - This control allows you to change the resolution, framerate and compression of the image stream to the dashboard, click the control to pop-up the configuration.
+#. Bandwidth Indicator - Indicates approximate bandwidth usage of the image stream. The indicator will display green for "safe" bandwidth usage, yellow when teams should use caution and red if the stream bandwidth is beyond levels that will work on the competition field.
+#. Framerate - Indicates the approximate received framerate of the image stream.
+
+Drive
+-----
+
+.. image:: images/driver-station-labview-dashboard/drive.png
+
+The center pane contains a section that provides feedback on the joysticks and drive commands when used with the LabVIEW framework and a section that displays the robot IP and reported battery voltage:
+
+#. Displays X,Y and Throttle information and button values for up to 2 joysticks when using the LabVIEW framework
+#. Displays values being sent to motor controllers when using LabVIEW framework
+#. Displays a brief chart of the reported battery voltage
+#. Displays a Gyro value
+
+These indicators (other than the Gyro) are hooked up to appropriate values by default when using the LabVIEW framework. For information on using them with C++/Java code see :doc:`using-the-labview-dashboard-with-c++-java-code`.
+
+Autonomous Selection Box and Connection Indicator
+-------------------------------------------------
+
+.. image:: images/driver-station-labview-dashboard/autonomous-selection-box-and-connection-indicator.png
+
+This year, the Dashboard also contains 2 new elements on the Drive tab:
```

Comment:
Can delete this line, as they are no longer new.

Reply:
I reworked this section to combine it with the section above so all information about the `Drive` tab stays together. Is that okay?